feat: stage 0 3d 도형 구현

yourchaee opened the pull request:
## 개요
스테이지 0 - 3d 도형 구현 

## 작업 내용
스테이지 0 - 3d 도형 구현 로직
기본 카메라 설정
기본 조명 설정
도형 회전 기능

## 특이 사항
N/A

yourchaee committed:
fix: 폴더 구조 및 파일 정리

- 기존 폴더 구조 변경
- 파일 위치 변경
- 불필요한 파일 삭제

yourchaee committed:
feat: cube 기본 세팅

- 3d 도형 구현을 위한 큐브 기본 세팅

yourchaee committed:
feat: 스테이지 0 3d 도형 구현

- 스테이지 0 3d 도형 구현 로직
- 기본 카메라 설정
- 기본 조명 설정

j25nkh commented on src/App.js:
```diff
@@ -0,0 +1,8 @@
+import "./reset.css";
+import StageZero from "./components/stages/StageZero";
```
import 의 순서가 바뀌면 좋을 것 같습니다.

j25nkh commented on src/components/stages/StageZero.js:
```diff
@@ -0,0 +1,43 @@
+
+export default function StageZero() {
+  const defaultColor = { r: 51, g: 153, b: 255 };
+  return (
```
return 위에 개행이 들어가면 일관성이 있겠네요

sageherb commented on src/components/stages/StageZero.js:
```diff
@@ -0,0 +1,43 @@
+        castShadow
+      />
+      <Physics>
+        {stageZeroCoordinates.positions.map((coordinate, id) => (
```
map 콜백 함수의 두 번째 인자는 index니까 id 말고 index가 더 어울릴 것 같아요!

sageherb commented on src/components/objects/Cube.js:
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
```
주석 제거해주세요!

j25nkh commented on src/components/stages/StageZero.js:
```diff
@@ -0,0 +1,43 @@
+  const defaultColor = { r: 51, g: 153, b: 255 };
+  return (
+    <Canvas
+      style={{ height: "100vh" }}
```
inline style의 사용은 자제해 주시기 바랍니다.

j25nkh reviewed (dismissed):
작은 부분들까지 신경써주시면 좋을 것 같습니다. 고생 하셨습니다.

yourchaee committed:
fix: 코드 리뷰 수정사항 변경

- 파일 import 순서 변경
- inline style css -> styled components 변경
- 개행
- eslint 설정 변경
- 불필요한 기본 세팅 값 삭제

yourchaee committed:
chore: eslint 디펜던시 삭제

yourchaee committed:
style: 변수명 변경

sageherb reviewed (approved):
고생하셨습니다!
작업 내용 자세히 써주시면 좋을 것 같아요!

j25nkh reviewed (approved):
기존 피드백 사항을 잘 반영해주셔서 추가 사항은 없습니다.
